src/styles/page/button.scss

```diff
@@ -0,0 +1,21 @@
+$color: rgba(0, 172, 220, 1);

```

Comment:
You have this color declared in different files.
Declare it once in file with variables and use where you need.